### ISSUE TITLE
[Request] PriceSpy

### ISSUE THREAD

**Issue by KnifeFed** [qsniyg/maxurl#930]
https://pricespy.co.uk/games-consoles/game-consoles/sony-playstation-5-ps5-825gb--p5113846

```
https://pricespy-*.kxcdn.com/product/*/800/*.jpg
                                       ^800 seems to be the highest it can go
```

This will work for all countries since they use the same CDN.

**Comment by qsniyg** [comment]
Thanks! I think the `75b8` part of `pricespy-75b8.kxcdn.com` is just the bucket ID for kxcdn.

**Comment by KnifeFed** [comment]
Right, but I figured the ID might change at some point in the future.

Thanks for implementing it!

**Comment by qsniyg** [comment]
>Right, but I figured the ID might change at some point in the future.

It's possible, but I'm guessing that would require the site buying another bucket, which might include organizational changes. I don't have particularly strong feelings either way, but I generally prefer to update the script to support new sites/domains as they arrive, rather than try to guess what their next move is. Overly general rules tend to cause more debugging problems than overly strict ones (though there is of course a balance to be had). It would be much easier had I implemented #12 sooner...
